.changeset/twelve-parents-glow.md

```diff
@@ -0,0 +1,72 @@
+---
+'@twilio-paste/alert': patch
+'@twilio-paste/alert-dialog': patch
+'@twilio-paste/anchor': patch
+'@twilio-paste/avatar': patch
+'@twilio-paste/badge': patch
+'@twilio-paste/base-radio-checkbox': patch
+'@twilio-paste/breadcrumb': patch
+'@twilio-paste/button': patch
+'@twilio-paste/button-group': patch
+'@twilio-paste/callout': patch
+'@twilio-paste/card': patch
+'@twilio-paste/chat-composer': patch
+'@twilio-paste/chat-log': patch
+'@twilio-paste/checkbox': patch
+'@twilio-paste/code-block': patch
+'@twilio-paste/combobox': patch
+'@twilio-paste/data-grid': patch
+'@twilio-paste/date-picker': patch
+'@twilio-paste/disclosure': patch
+'@twilio-paste/display-heading': patch
+'@twilio-paste/file-picker': patch
+'@twilio-paste/form-pill-group': patch
+'@twilio-paste/heading': patch
+'@twilio-paste/help-text': patch
+'@twilio-paste/in-page-navigation': patch
+'@twilio-paste/inline-code': patch
+'@twilio-paste/inline-control-group': patch
+'@twilio-paste/input': patch
+'@twilio-paste/input-box': patch
+'@twilio-paste/label': patch
+'@twilio-paste/list': patch
+'@twilio-paste/minimizable-dialog': patch
+'@twilio-paste/modal': patch
+'@twilio-paste/pagination': patch
+'@twilio-paste/paragraph': patch
+'@twilio-paste/popover': patch
+'@twilio-paste/radio-button-group': patch
+'@twilio-paste/radio-group': patch
+'@twilio-paste/screen-reader-only': patch
+'@twilio-paste/select': patch
+'@twilio-paste/separator': patch
+'@twilio-paste/side-modal': patch
+'@twilio-paste/skeleton-loader': patch
+'@twilio-paste/spinner': patch
+'@twilio-paste/switch': patch
+'@twilio-paste/table': patch
+'@twilio-paste/tabs': patch
+'@twilio-paste/textarea': patch
+'@twilio-paste/time-picker': patch
+'@twilio-paste/toast': patch
+'@twilio-paste/tooltip': patch
+'@twilio-paste/truncate': patch
+'@twilio-paste/aspect-ratio': patch
+'@twilio-paste/flex': patch
+'@twilio-paste/grid': patch
+'@twilio-paste/media-object': patch
+'@twilio-paste/stack': patch
+'@twilio-paste/box': patch
+'@twilio-paste/sibling-box': patch
+'@twilio-paste/text': patch
+'@twilio-paste/customization': patch
+'@twilio-paste/design-tokens': patch
+'@twilio-paste/data-visualization-library': patch
+'@twilio-paste/style-props': patch
+'@twilio-paste/website': patch
+'@twilio-paste/core': patch
+---
+
+Update packages to be ESM-compatible
+
+[Design Tokens] Update `tokens.generic.js` export from commonjs to ESM
```

Comment:
suggestion: I'd pull design tokens out of this changeset, and create its own changeset file, with this line as it's entry. Otherwise, every changelog for the packages listed will get this line about design tokens added